README.md

```diff
@@ -10,9 +10,6 @@ This has only been tested using:
 ## Pre-requisites
 A working [Docker](https://www.docker.com/community-edition#/download) installation on the platform of choice.
 
-## Note
-You cannot use docker exec -it osehravista bash to get access to the mumps prompt. This is likely due to how docker permission schemes for shared memory access works. Instead always gain access via the ssh commands below
```

Comment:
Why is this deleted? I've verified that the issue still exists, though I have more information: This only occurs if someone else is accessing the instance at the same time (CPRS, etc.) as you do a docker exec -it osehravista bash. The workaround is to ssh into the instance after running the docker exec -it osehravista command.